src/lib.rs

```diff
@@ -62,6 +70,14 @@ impl Builder {
         self
     }
 
+    /// Whether to show all categories by default (versus only those that are explicitly enabled).
+    ///
+    /// Defaults to true.
+    pub fn show_all_categories(mut self, show_all_categories: bool) -> Self {
+        self.show_all_categories = show_all_categories;
+        self
+    }
```

Comment:
It might be good to suggest in the Documentation to use `enable_categories_button(false)` when setting this to false.